src/components/Canvas/Canvas.vue

```diff
@@ -118,8 +110,8 @@ export default {
   },
 
   async mounted () {
-    const config = await fetch(window.__TCV_CONFIG.configPath)
-    this.config = await config.json()
+    this.config = JSON.stringify(__TAILWIND_CONFIG__)
+    this.config = JSON.parse(this.config)
```

Comment:
Here's where vite gives us the stringified tailwind config as part of the build instead of us fetching it from the separate server. I tried to do the parsing and stringifying in one line but I was seeing unexpected things when I logged to the console - I'm not sure why :shrug: